diff --git a/package.json b/package.json
index 93dd69e..3222a8e 100644
--- a/package.json
+++ b/package.json
@@ -22,7 +22,7 @@
     "@types/jest": "^27.5.0",
     "@vscode/test-web": "^0.0.24",
     "@vuedx/monorepo-tools": "^0.2.2-next-1651055813.0",
-    "esbuild": "^0.14.38",
+    "esbuild": "^0.12.15",
     "husky": "^7.0.4",
     "jest": "^27.0.0",
     "lint-staged": "^12.4.1",
diff --git a/packages/grammarly-languageserver/src/services/CodeActionService.ts b/packages/grammarly-languageserver/src/services/CodeActionService.ts
index f133dcc..0ec1baf 100644
--- a/packages/grammarly-languageserver/src/services/CodeActionService.ts
+++ b/packages/grammarly-languageserver/src/services/CodeActionService.ts
@@ -93,7 +93,6 @@ export class CodeActionService implements Registerable {
         suggestionId,
         replacementId,
       })
-      console.log(JSON.stringify(edit, null, 2))
       const range = document.findOriginalRange(edit.range.start, edit.range.end)
       const newText = document.toText(edit.content)
 
