diff --git a/node_modules/grammarly-languageserver/dist/index.node.cjs b/node_modules/grammarly-languageserver/dist/index.node.cjs
index 510bb2f..d0b22fe 100644
--- a/node_modules/grammarly-languageserver/dist/index.node.cjs
+++ b/node_modules/grammarly-languageserver/dist/index.node.cjs
@@ -198,7 +198,6 @@ let DocumentService = class DocumentService {
                 __classPrivateFieldGet(this, _DocumentService_onDocumentOpenCbs, "f").forEach((cb) => cb(document));
             }),
             __classPrivateFieldGet(this, _DocumentService_documents, "f").onDidClose(({ document }) => {
-                console.log('close', document.original.uri);
                 __classPrivateFieldGet(this, _DocumentService_onDocumentCloseCbs, "f").forEach((cb) => cb(document));
                 document.session.disconnect();
             }),
@@ -536,7 +535,6 @@ let CodeActionService = class CodeActionService {
                 suggestionId,
                 replacementId,
             });
-            console.log(JSON.stringify(edit, null, 2));
             const range = document.findOriginalRange(edit.range.start, edit.range.end);
             const newText = document.toText(edit.content);
             codeAction.edit = {
